dev/docs/builtins-reference.md

```diff
@@ -291,34 +291,62 @@ y = X %*% rand(rows=ncol(X), 1)
 [C, S] = steplm(X = X, y = y, icpt = 1);
 ```
 
-## `slicefinder`-Function
+## `outlier-Function
 
```

Comment:
It seems `slicefinder` has been removed may be we should not.

Reply:
Actually I was not getting the correct format for outlier so I just replaced it. But don't worryi will add slicefinder again.